frontend/src/scenes/feature-flags/FeatureFlagSnippets.tsx

```diff
@@ -1,130 +1,327 @@
 import { useValues } from 'kea'
 import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
 import { teamLogic } from 'scenes/teamLogic'
+import { GroupType } from '~/types'
 
 export const UTM_TAGS = '?utm_medium=in-product&utm_campaign=feature-flag'
 
-export function NodeJSSnippet({ flagKey }: { flagKey: string }): JSX.Element {
-    return (
-        <>
-            <CodeSnippet language={Language.JavaScript} wrap>
-                {`const isMyFlagEnabledForUser = await client.isFeatureEnabled('${flagKey}', 'user distinct id')
+export interface FeatureFlagSnippet {
+    flagKey: string
+    multivariant?: boolean
+    groupType?: GroupType
+    localEvaluation?: boolean
+    payload?: boolean
+}
+
+export function NodeJSSnippet({
+    flagKey,
+    groupType,
+    multivariant,
+    localEvaluation,
+    payload,
```

Comment:
nit: I think forcing all these options into one code snippet definitely makes it more unreadable and harder to parse

Reply:
so I don't love it but if it works then 👍

Reply:
Agree it's a trade-off, although I prefer this for maintainability + extending things.